assets/armsnippets.jsonc

```diff
@@ -1715,7 +1715,7 @@
             "{",
             "\t\"name\": \"${1:logicApp1}\",",
             "\t\"type\": \"Microsoft.Logic/workflows\",",
-            "\t\"apiVersion\": \"2019-05-01\",",
+            "\t\"apiVersion\": \"2017-07-01\",",

```

Comment:
This will cause my work-around to fail.  Could you run the "Logic App" suite (via launch.json) and make sure it's fixed?  Thanks!